pylti1p3/contrib/flask/cookie.py

```diff
@@ -0,0 +1,31 @@
+from pylti1p3.cookie import CookieService
+
+
+class FlaskCookieService(CookieService):
+    _request = None
+    _cookie_data_to_set = None
+
+    def __init__(self, request):
+        self._request = request
+        self._cookie_data_to_set = {}
+
+    def _get_key(self, key):
+        return self._cookie_prefix + '-' + key
+
+    def get_cookie(self, name):
+        return self._request.get_cookie(self._get_key(name))
+
+    def set_cookie(self, name, value, exp=3600):
+        self._cookie_data_to_set = {
+            'key': self._get_key(name),
+            'value': value,
+            'exp': exp
+        }
+
+    def update_response(self, response):
+        if self._cookie_data_to_set:
+            response.set_cookie(
```

Comment:
To avoid issues with execution LTI launch in iframes will be good to set cookies as `samesite=None` + `secure=True` in case if request is HTTPS. Please check my solution for Django: https://github.com/dmitry-viskov/pylti1.3/blob/master/pylti1p3/contrib/django/cookie.py#L47

Also I may suggest to set cookies as `httpomly`

Reply:
Httponly flag added

Reply:
@dmitry-viskov 
Also added docs. Please take a look and let me know it you think that flow is ok for you.

Reply:
@dmitry-viskov I updated cookie with secure, httponly and samesite

Reply:
Looks good to me! 👍 

As i may see from this [issue](https://github.com/pallets/werkzeug/issues/1549) `samesite=None` is supported only from `Werkzeug==1.0.0`. I may be wrong but looks like there could be an error in case of `None` value: https://github.com/pallets/werkzeug/pull/1550/files#diff-6deb2bc55f24875e528da32c81c57c2cL1159
?
So i may suggest to check `Werkzeug` version before `set_cookie` and probably for the versions early than `1.0.0` add headers using workaround:
```
resp.headers.add('Set-Cookie', 'cookie-key=cookie-value; SameSite=None; Secure')
```
Or we may just ignore this for now and add this workaround later in the future versions of the library.

Reply:
Done

When using version before `1.0.0` samesite is not set. I suppose setting cookie value by hand may be tricky and involve some more logic.